site/content/usage/06-vpc-networking.md

```diff
@@ -160,3 +160,29 @@ See the complete example [here](https://github.com/weaveworks/eksctl/blob/master
 
 **Note**: Specifying the NAT Gateway is only supported during cluster creation and it is not touched during a cluster
 upgrade. There are plans to support changing between different modes on cluster update in the future.
+
+### Managing Access to the Kubernetes API Server Endpoints
+
+The default creation of an EKS cluster exposes the Kubernetes API server publicly but not directly from within the
+VPC subnets (Public=true, Private=false). Traffic destined for the API server from within the VPC
+must first exit
+the VPC networks (but not Amazon's network) and then re-enter to reach the API server. Allowing
+direct access from
+within the VPC subnets is possible but not without some caveats.
+See [Amazon EKS Cluster
+Endpoint Access Control](https://docs.aws.amazon.com/eks/latest/userguide/cluster-endpoint.html) for
+further information on additional steps needed when public=false, private=true.
+
+The Kubernetes API server endpoint access for a cluster can be cofigured for public and private
+access by specifiying
+the `--public-endpoint-access` and/or `--private-endpoint-access` CLI flags to be `true` (access
```

Comment:
my first thought (without looking up how we name other flags/clusterconfigs) would be to do this the same as the cluster config path which would be `--vpc-endpoint-access-{public,private}`

Reply:
> documentation is easy to understand 👍 I would want to know what the default is, though.
> 
> Edit: or is the default going to stay as is (Public=true, Private=false)?

That's a good question.  The submitter of #649 argues for making the default in eksctl to public = true, private = true, which seems like a rational choice.  I wonder though if those using eksctl today might be caught unaware by the default behavior (where eksctl uses the default EKS settings for this) changing to this new default.

Reply:
> my first thought (without looking up how we name other flags/clusterconfigs) would be to do this the same as the cluster config path which would be `--vpc-endpoint-access-{public,private}`

I've made this change as it appears to be consistent with other CLI flags.